python-threatexchange/threatexchange/signal_type/tests/test_raw_text.py

```diff
@@ -8,16 +10,19 @@
 class TestRawTextSignal(MatchesStrAutoTest):
     TYPE = RawTextSignal
 
-    def get_validate_hash_cases(self):
+    @pytest.fixture
+    def validate_hash_cases(self):
```

Comment:
blocking: This doesn't look like it needs to be a fixture - it looks like a better fit for pytest.mark.parametrize

Reply:
Any response on this unadressed blocking comment?